app/src/main/java/io/homeassistant/companion/android/migrations/Migrations.kt

```diff
@@ -14,7 +14,7 @@ class Migrations constructor(
         private const val TAG = "Migrations"
         private const val PREF_NAME = "migrations"
         private const val PREF_VERSION = "migration_version"
-        private const val LATEST_VERSION = 3
+        private const val LATEST_VERSION = 4
```

Comment:
This version just happened to match the DB version, if you aren't adding a non database migration you don't need to increment.

Reply:
Got it. Fixed.